napari/_qt/widgets/qt_viewer_buttons.py

```diff
@@ -7,13 +7,11 @@
 from qtpy.QtWidgets import (
     QApplication,
     QDoubleSpinBox,
-    QFormLayout,
     QFrame,
     QGridLayout,
     QHBoxLayout,
     QLabel,
     QPushButton,
-    QVBoxLayout,
```

Comment:
It's weird to me that this appears as a change, but I checked the file and it's correct, so 🤷. Maybe a GH or pre-commit quirk.